### PR TITLE
Development_Setup.rst: Change link address

### PR DESCRIPTION
**Pull request by Mohit17067**
This changes the redirecting link to installing
from git in 'Using venv' section.

Closes https://github.com/coala/coala/issues/5001

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by nvzard** [comment]
LGTM 
ack b4df49b

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Using venv
         (coala-venv)$ deactivate
 
 - Now you can activate the environment and start
-  `the next part <https://coala.io/devsetup#installing-from-git>`_.
+  `the next part <https://api.coala.io/en/latest/Developers/Development_Setup.html#id1>`_.
```
Comment (Makman2) [inline docs/Developers/Development_Setup.rst]: you don't need `#id1` :+1:

Comment (Makman2) [inline docs/Developers/Development_Setup.rst]: And actually, it's linking to this document again :O

**Comment by Makman2** [comment]
unack b4df49b

File: docs/Developers/Development_Setup.rst (hunk 1 above)
Comment (Mohit17067) [reply docs/Developers/Development_Setup.rst]: With \#id1 it is linking to the required link. Could you please explain why we do not need it.

Comment (Makman2) [reply docs/Developers/Development_Setup.rst]: ah sorry, last time it didn't jump to the paragraph in my broswer (probably I was too impatient and scrolled). Nvm :+1:

**Comment by Makman2** [comment]
ack b4df49b

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**